colibris/views.py

```diff
@@ -179,10 +199,28 @@ def get_schema(self, *args, **kwargs):
 
         return json_payload
 
+    async def get_validated_query(self, schema=None):
+        if schema is None:
+            schema = self.get_query_schema_class()
+
+        assert schema is not None, 'The attribute "query_schema_class" is required for {}'.format(self)
+
+        json_payload = await self.get_request_query()
```

Comment:
A better name for this local var would be `query`. We should not assume it's a JSON and it's not really a payload *per se*.

Reply:
Sure! May bad.